src/ui/propertiesdialog.cpp

```diff
@@ -384,25 +384,26 @@ void CAPropertiesDialog::createDocumentFromTree() {
 
 		for ( int j=0; j<cur->child(i)->childCount(); j++ ) {
 			CAContext *c = _contextItem[cur->child(i)->child(j)];
-			switch ( c->contextType() ) {
-				case CAContext::Staff: {
-					CAStaff *s = static_cast<CAStaff*>(c);
-					while ( s->voiceList().size() )
-						s->removeVoice( s->voiceList()[0] );
-
-					for ( int k=0; k<cur->child(i)->child(j)->childCount(); k++ ) {
-						s->addVoice( _voiceItem[cur->child(i)->child(j)->child(k)] );
-					}
-				}
-				default:
-					sheet->addContext(c);
-					break;
+			if( c->contextType() == CAContext::Staff )
+            {
+                CAStaff *s = static_cast<CAStaff*>(c);
+                while ( s->voiceList().size() )
+                    s->removeVoice( s->voiceList()[0] );
+
+                for ( int k=0; k<cur->child(i)->child(j)->childCount(); k++ ) {
+                    s->addVoice( _voiceItem[cur->child(i)->child(j)->child(k)] );
+                }
+            }
+            else
+            {
+                sheet->addContext(c);
+                break;
```

Comment:
When implemented chord names I decided to use swtich-case blocks when having logic based on the derived class type. Is there a reason why reverting back to if-else?
PS: Also fix indentation, if you think if-else is the right way.